BatchAuctionOptimization/batchauctions.tex

```diff
@@ -557,30 +524,48 @@ \section{Models and Formulations}
   \begin{array}{rll}
     x &= & 0 \\
     y &= & 0 \\[1mm]
-    \frac{p_j}{p_k} &> & \pi
+    \frac{p_j}{p_k} &< & \pi
   \end{array}
   \right\}
   \label{eq:order_model_generic}
 \end{align}
 
 \paragraph{Objective}
 
-Our primary objective shall be to determine token prices/exchange rates such that the feasible
-\emph{trading volume} is maximized.
+Our primary objective shall be to determine token prices/exchange rates such that the total
+feasible \emph{trading volume} is maximized.
 Thereby, trading volume refers to the sum of all tokens traded across all orders, weighted by their
 respective price, \ie it measures the value of all transactions in terms of units of the reference
 token $ \tau_0 $.
-Thus, for every order $ \omega = (j,k,\overline{x},\overline{y},\pi) $, we introduce an additional
+Thus, for every order $ \omega = (j,k,\overline{y},\pi) $, we introduce an additional
 variable $ v \ge 0 $, for which we require
 \begin{align}
-  v = x \cdot p_j \qquad \text{and} \qquad v = y \cdot p_k
+  v = y \cdot p_j \qquad \text{and} \qquad v = x \cdot p_k
   \label{eq:trading_volume}
 \end{align}
 The trading volumes of all orders shall be stored in a vector
 $ \mathbf{v} \in \mathbb{R}^N_{\ge 0} $.
```

Comment:
This should be on the previous line (one sentence per line)

Reply:
My supervisor at university preferred at most 100 characters per line, so my text editor breaks lines in the middle of sentences :-)
Don't like that such much myself but stuck to it - should probably change this at some point (would change the entire doc)

Reply:
Sounds like too much. 100 chars per line is a standard for code, never really thought about it for tex source files (since they aren't meant to be read anyway).